motion_control/engines/platform_engine/Makefile.dep

```diff
@@ -1,4 +1,5 @@
 USEMODULE += motion_control_common
```

Comment:
In `platform engine: replace callbacks by object references`, the scope is not good, should be:
`motion_control/engines: replace callbacks by object references`